rules/build_sector.smk

```diff
@@ -466,10 +466,12 @@ def input_hera_data(w) -> dict[str, str]:
         result = {}
         for year in unique_years:
             result[f"hera_river_discharge_{year}"] = (
+                # rules.retrieve_hera_data.output['river_discharge']
                 f"data/hera_{year}/river_discharge_{year}.nc"
             )
             result[f"hera_ambient_temperature_{year}"] = (
-                f"data/hera_{year}/ambient_temp_{year}.nc"
+                # rules.retrieve_hera_data.output['ambient_temperature']
```

Comment:
The commented-out version looks correct, the active one doesn't. Was this intentional?